www/includes/easyparliament/templates/html/static/help.php

```diff
@@ -4,264 +4,396 @@
+                    Use TheyWorkForYou as a gateway, rather than a simple place to find a number
+                    measuring competence.</p>
+
+                    <p>If you have suggestions for other metrics we could add which would be
+                    useful, send them to the <a href="mailto:<?=str_replace('@', '&#64;', CONTACTEMAIL) ?>">usual
+                    address</a>. We&rsquo;ve got a few ideas ourselves, to keep you on your toes.</p>
+
+                </dd>
+
+                <dt id="pps_unknown">
+                    Why are the PPS positions out of date?
+                </dt>
+                <dd>
+                    <p>TheyWorkForYou used to get its ministerial information, as with much else,
+                    by scraping various pages on the official parliament.uk site. The official site
+                    stopped maintaining a list of PPSs in January 2009, which means our scraper could
+                    no longer find and work out the information. Without a central list of
+                    Parliamentary Private Secretaries, there isn&rsquo;t a lot we can do.</p>
+                </dd>
+
+                <dt id="dates_wrong">
+                    The date someone starts/ends a ministerial position is wrong!
+                </dt>
+                <dd>
+                    <p>TheyWorkForYou gets its current ministerial position information from
+                    <a href="http://data.parliament.uk/membersdataplatform/memberquery.aspx">Parliament&rsquo;s official API</a>,
+                    so for any issues with current dates, please contact Parliament. We used to
+                    obtain data by scraping various pages on the official parliament.uk site, so
+                    there might be issues on our site with older data. We hope to also fetch old
+                    ministerial information from Parliament&rsquo;s API in future, which should hopefully
+                    fix any outstanding issues.</p>
+                </dd>
+
+                <dt id="independent">
+                    What is an Independent, or Independent &lt;party&gt; MP?
+                </dt>
+                <dd>
+                    <p>If an MP is a member of a local, perhaps one-issue, party, or is not a
+                    member of any political party, TheyWorkForYou lists them as Independent, e.g.
+                    <a href="/mp/richard_taylor/wyre_forest">Richard Taylor</a> or
+                    <a href="/mp/robert_wareing/liverpool,_west_derby">Robert Wareing</a>.
+                    If an MP does not take the party whip, but is a member of that party, we list them as either
+                    Independent followed by the party name, e.g. <a href="/mp/clare_short/birmingham,_ladywood">Clare Short</a>.
+                </p>
+
+                </dd>
+            </dl>
 
-            <dt><a name="use"></a>How do people use TheyWorkForYou?</dt>
```

Comment:
Not sure if this is going to go elsewhere, but I point people at this question relatively frequently – I think it's an important one to have somehow.